qdrant_client/conversions/common_types.py

```diff
@@ -62,16 +62,16 @@ def get_args_subscribed(tp):  # type: ignore
     rest.PayloadSchemaType, rest.PayloadSchemaParams, int, grpc.PayloadIndexParams
 ]  # type(grpc.PayloadSchemaType) == int
 PointStruct: TypeAlias = rest.PointStruct
-Points = Union[rest.Batch, List[Union[rest.PointStruct, grpc.PointStruct]]]
+Points = Union[rest.Batch, Sequence[Union[rest.PointStruct, grpc.PointStruct]]]
```

Comment:
`LocalCollection.upsert` should become `Union[Sequence[models.PointStruct], models.Batch]` then